ihmc-high-level-behaviors/src/main/java/us/ihmc/PerceptionAndAutonomyProcess.java

```diff
@@ -567,6 +594,51 @@ public void updatePlanarRegions()
       }
    }
 
+   public void updateHeightMap()
+   {
+      if (realsenseDepthImage != null && realsenseDepthImage.isAvailable() && heightMapDemandNode.isDemanded())
+      {
+         RawImage latestRealsenseDepthImage = realsenseDepthImage.get();
+
+         if (rapidHeightMapExtractor == null)
+         {
+            rapidHeightMapExtractor = new RapidHeightMapExtractor(openCLManager, null); // TODO: Fix up for resetting to feet Z
```

Comment:
The only thing the height map needs the reference frames for is to reset the height based on mid feet Z (or I think we might want lowest foot Z) Anyway, it's happy with null and we'll need to add a UI button somewhere to reset it.

Reply:
Nice! Since this is assumed to be flat ground it won't matter which foot. One of the future tasks I have is to reset the height map to the lowest foot (I'm trying to leave some easy tasks for future interns which is why I haven't done this)